webpack.config.js

```diff
@@ -55,7 +58,82 @@ if (process.env.TEST_ENV) {
   displayIcons = icons.slice(0, 255);
 }
 
-module.exports = (env, argv) => {
+const pageDescription = `${icons.length} Free SVG icons for popular brands.`,
+  pageTitle = 'Simple Icons',
+  pageUrl = 'https://simpleicons.org',
+  logoUrl = 'https://simpleicons.org/icons/simpleicons.svg';
+
+async function generateStructuredData() {
+  const getSimpleIconsMembers = new Promise(async (resolve, reject) => {
+    const siMembersCacheFilePath = path.join(
+      os.tmpdir(),
+      'simple-icons-members.json',
+    );
+    if (fs.existsSync(siMembersCacheFilePath)) {
+      try {
+        const siMembersFileContent = fs.readFileSync(siMembersCacheFilePath, {
+          encoding: 'UTF8',
+        });
+        resolve(JSON.parse(siMembersFileContent));
+      } catch (error) {
+        reject(error);
+      }
```

Comment:
```js
const getSimpleIconsMembers = async () => {
  const siMembersCacheFilePath = path.join(
    os.tmpdir(),
    'simple-icons-members.json'
  );
  if (fs.existsSync(siMembersCacheFilePath)) {
    const siMembersFileContent = fs.readFileSync(siMembersCacheFilePath, {
      encoding: 'UTF8',
    });
    return JSON.parse(siMembersFileContent);
  } else {
    // ...
  }
};
```
@mondeja sorry, should have been more clear. I meant we can do something like the following instead of using all the promise wrappers.

Reply:
And what about the other `resolve`? Is inside a `.then` block because `Promise.all` is being used.

Feel free to push the change that you are thinking directly in the PR 👍🏼

Reply:
pushed the changes